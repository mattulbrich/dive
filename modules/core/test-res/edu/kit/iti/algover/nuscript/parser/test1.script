/* This is a script test */

testFile on='someTerm' string="someString" boolean=true int=22;

cases {
   case "case 1":
      z3;
   case "case 2":
      // remains open
}