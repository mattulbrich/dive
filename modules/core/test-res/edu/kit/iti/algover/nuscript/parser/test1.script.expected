testFile on='someTerm' string="someString" boolean=true int=22;
cases {
  "case 1":
    z3;
  "case 2":
}